MFVNeutralino/python/AnalysisCuts_cfi.py

```diff
@@ -13,26 +17,31 @@
                                apply_presel = apply_presel,  # 1 = jets, 2 = el/mu, 3 = jets OR bjet/displaced dijet triggers, 4 = bjet/displaced dijet triggers veto HT trigger, 5 = MET trigger
                                                              # 6 = bjets/displaced dijet (HT agnostic)
                                require_met_filters = cms.bool(True) if use_MET_triggers else cms.bool(False),
-                               require_bquarks  = cms.bool(False),
-                               require_trigbit  = cms.bool(True),
+                               require_bquarks  = cms.bool(True) if use_btag_triggers else cms.bool(False),
+                               require_trigbit  = cms.bool(True) if use_btag_triggers else cms.bool(False),
```

Comment:
I'm confused about this one too (perhaps also for @shaunhogan). If it's strictly for studying the trigger behavior, we can set it based on different flags. https://github.com/DisplacedVertices/cmssw-usercode/blob/UL_V30_BjetDispljet/MFVNeutralino/plugins/AnalysisCuts.cc#L482 makes me think it also pertains to any time that we require that the trigger fired (which, other than for specific trigger studies), should be everywhere in the analysis! 

Also, a general question for @abbywarden and @pkotamnives now that I look closer: Does the AnalysisCuts.cc that I linked to currently include the lepton triggers at all? Maybe I just need to make it to the changes to that file in this PR to get the answer! 😀